app/helpers/groups_helper.rb

```diff
@@ -0,0 +1,26 @@
+module GroupsHelper
+    
+    def first_choice
+        if @group[:first_choice].nil?
+            "not chosen"
+        else
+            @group[:first_choice]
+        end
+    end
+    
+    def second_choice
+        if @group[:second_choice].nil?
+            "not chosen"
+        else
+            @group[:second_choice]
+        end
+    end
+    
+    def third_choice
+        if @group[:third_choice].nil?
+            "not chosen"
+        else
+            @group[:third_choice]
+        end
+    end
```

Comment:
Same with these. Can we get one central method and then call on it repeatedly? This to me looks like messy code that can be simplified down